docs/2.0/integration/testng.adoc

```diff
@@ -167,23 +167,36 @@ They can: be parametrized, make checks, have nested steps, and create attachment
 
 In order to define steps in Java code, you need to annotate the respective methods with **@Step** annotation.
 When not specified, the step name is equal to the annotated method name converted to human-readable format.
-To define an explicit step name, pass your own **String** argument:
+
+Note that steps' mechanics was revised and now it supports smart fields' analysis.
```

Comment:
That is great. However, will it still support the previous position base approach? If so, that would be nice to have explicitly stated.